javascript/index.js

```diff
@@ -39,27 +48,46 @@ function clearSplits() {
 }
 
 function setStopBtn() {
-  // ... your code goes here
+  btnLeft.className = 'btn stop';
+  btnLeft.innerText = 'STOP'; // ... your code goes here
 }
 
 function setSplitBtn() {
-  // ... your code goes here
+  btnRight.className = 'btn split';
+  btnRight.innerText = 'SPLIT';// ... your code goes here
 }
 
 function setStartBtn() {
-  // ... your code goes here
+  btnLeft.className = 'btn start';
+  btnLeft.innerText = 'START'; // ... your code goes here
 }
 
 function setResetBtn() {
-  // ... your code goes here
+  btnRight.className = 'btn reset';
+  btnRight.innerText = 'RESET'; // ... your code goes here
 }
 
 // Start/Stop Button
 btnLeft.addEventListener('click', () => {
   // ... your code goes here
+
+  // alert('Hello Katie');
+  if (btnLeft.classList.contains('start') === true) {
+    chronometer.startClick(() => {
+      printTime();
+    });
```

Comment:
very clever using an anonymous function that will run printTime, however, just passing printTime as an argument would work just as well.